must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -156,20 +155,22 @@ for ns in $namespaces; do
         done
 
         # Collecting snapshot information for ceph subvolumes
-        printf "collecting snapshot info for ceph subvolumes \n" | tee -a "${BASE_COLLECTION_PATH}"/gather-debug.log
-        COMMAND_OUTPUT_FILE=${COMMAND_OUTPUT_DIR}/cephfs_snap_info.log
+        printf "collecting snapshot info for cephFS subvolumes \n" | tee -a "${BASE_COLLECTION_PATH}"/gather-debug.log
+        COMMAND_OUTPUT_FILE=${COMMAND_OUTPUT_DIR}/cephfs_subvol_and_snap_info
 
         # Inspecting CephFS filesystems
         filesystems=$(oc get cephfilesystems.ceph.rook.io -n openshift-storage -o jsonpath="{range .items[*]}{@.metadata.name}{'\n'}{end}")
         for fs in $filesystems; do
```

Comment:
define a variable for csi group name it will be easy to replace it later if we start supporting multiple subvolume groups

Reply:
Didn't add it because in that case we would have to run one more loop and whole logic would change.

Reply:
agreed. @agarwal-mudit can create var with csi

Reply:
don't add a loop . just a var

Reply:
I can do it but not required because if we support multiple subvolumegroups we need a complete different logic where we would have to find all the subvolumegroups and then extract all the information for each subvolumegroup one by one. And thus whole logic which is there today would change. Let me know if you guys still want me to add the variable

Reply:
Yes, correct we need to first get all group in a volume and then check subvolumes in each group.

how about below logic we need to add one more for loop for groups in a volume
```
filesystems=$(oc get cephfilesystems.ceph.rook.io -n openshift-storage -o jsonpath="{range .items[*]}{@.metadata.name}{'\n'}{end}")
        for fs in $filesystems; do
           subvolgroups=$(oc -n openshift-storage exec "${HOSTNAME}"-helper -- bash -c "ceph fs subvolumegroup ls $fs | jq --raw-output '.[].name' ")
           for group in $subvolgroups:
               ---
```
if we don't it we don't need to hardcode CSI right?

Reply:
Fixed

Reply:
I still see the same code, missed to push the code?

Reply:
Yeah, testing it after the change.

Not adding the loop as of now, just added the variable.

Reply:
pushed now, PTAL